.travis.yml

```diff
@@ -55,6 +55,11 @@ _ios_job_template: &ios_job_template
     - export PATH=`pwd`/flutter/bin:`pwd`/flutter/bin/cache/dart-sdk/bin:$PATH
   script:
     - cd example
+    - flutter pub upgrade
```

Comment:
gotcha, I was just playing around to make it pass somehow anyways ;)

Reply:
By the way, do you happen to have any better idea why it fails on CI environment, 
I don't see a reason for it not to work, but it doesn't seem to use the correct updated version of permission_handler package, (upgraded to have more sound nullsafety, also bugfixes were taken place since the previous version)
I'm on Xcode 12.4, flutter 2.0.4.

Reply:
Maybe CI is running outdated XCode?

Reply:
can easily be that, shall i ammend it or will address it?

Reply:
If you could, please. 🙂  This line: https://github.com/Polidea/FlutterBleLib/blob/develop/.travis.yml#L50

Reply:
will do